modules/002-deckhouse/docs/FAQ.md

```diff
@@ -30,6 +30,20 @@ Then you can check report:
 kubectl logs job.batch/kube-bench
 ```
 
+{% offtopic title="**Caution.** Warning: Log Retention Settings Discrepancy" %}
+
+When deploying a Kubernetes cluster using Deckhouse, please be aware of the log retention settings. By default, Deckhouse sets the log retention period to 7 days. However, according to the security requirements specified in kube-bench, logs should be retained for at least 30 days.
+
+Recommendation:
+
+* Pay attention to potential discrepancies with kube-bench requirements when configuring log retention parameters.
+
+* If you decide to extend the log retention period beyond 7 days, it is recommended to use separate storage for logs. This will help avoid potential performance issues and ensure data preservation.
```

Comment:
```suggestion
* If you need to keep logs for more than 7 days, use a separate storage for logs. This will help avoid potential performance issues and ensure data preservation.
```